packages/vuetify/src/directives/touch/__tests__/touch.spec.ts

```diff
@@ -0,0 +1,119 @@
+// Directives
+import Touch from '../'
+
+// Libraries
+import Vue from 'vue'
+
+// Utilities
+import {
+  createLocalVue,
+  mount,
+  Wrapper
+} from '@vue/test-utils'
+
+// Simulate touch
+function touch (element) {
+  const createTrigger = eventName => (clientX, clientY) => {
+    const touches = [{ clientX, clientY }]
+    const event = new Event(eventName)
+
+    event.touches = touches
+    event.changedTouches = touches
+    element.element.dispatchEvent(event)
+
+    return touch(element)
+  }
+
+  return {
+    start: createTrigger('touchstart'),
+    move: createTrigger('touchmove'),
+    end: createTrigger('touchend')
+  }
+}
+
+describe('touch.ts', () => {
+  let mountFunction: (value?: object) => Wrapper<Vue>
+  let localVue: typeof Vue
+
+  beforeEach(() => {
+    localVue = createLocalVue()
+
+    mountFunction = (value = {}) => {
+      return mount(localVue.component('test', {
```

Comment:
Why do we need `localVue.component` here?

Reply:
I just thought it would be a good practice to have independent copies of Vue for testing. I can replace it with Vue.

Reply:
No need for vue.component imho